src/OSPSuite.Presentation/Presenters/SimulationPredictedVsObservedChartPresenter.cs

```diff
@@ -55,22 +59,25 @@ protected override void UpdateAnalysis()
          if (!_identityRepositories.Any())
             return;
 
-         if (ChartIsBeingCreated)
+         //The chart will not come into initialization if no observed data was available
+         if (ChartIsBeingCreated || !_axesInitialized)
+         {
             _predictedVsObservedChartService.ConfigureAxesDimensionAndTitle(observationColumns, Chart);
```

Comment:
this line of code IS NEVER CALLED in MOBI for a new simulation because of line 50 that exit. Then the ChartIsCreated. O thought this would fix the issue , and I think it did but somehow I could not make it work again.

Reply:
Maybe I just need a pair of fresh eyes